samples/SocialSample/Startup.cs

```diff
@@ -45,6 +45,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
                 options.LoginPath = new PathString("/login");
             });
 
+            //app.UseFacebookAuthentication(app.GetOptions<FacebookOptions>()));
```

Comment:
Oops yeah I'll remove

On Sep 11, 2015, at 10:23 AM, Chris R <notifications@github.com<mailto:notifications@github.com>> wrote:


In samples/SocialSample/Startup.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2faspnet%2fSecurity%2fpull%2f441%23discussion_r39294307&data=01%7c01%7chaok%40microsoft.com%7c620df78cee514b960a8708d2bacdbcf3%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=oTqI7xNpED2vKfnhlBqVh7SD%2bgIPSGbyXBLmBGMb4cQ%3d>:

> @@ -45,6 +45,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
>                  options.LoginPath = new PathString("/login");
>              });
>
> +            //app.UseFacebookAuthentication(app.GetOptions<FacebookOptions>()));


?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2faspnet%2fSecurity%2fpull%2f441%2ffiles%23r39294307&data=01%7c01%7chaok%40microsoft.com%7c620df78cee514b960a8708d2bacdbcf3%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=WepOsGfG1RAOONJhRGt2briCeeZz5YgKaowTzK2cP7g%3d>.